README.md

```diff
@@ -1,24 +1,58 @@
-# README
+# Recipe-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- ROR
+- Postgres
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+You must have Ruby installed in your computer.
 
-* How to run the test suite
+### Clone this repo
 
-* Services (job queues, cache servers, search engines, etc.)
+`https://github.com/Tadesse-Alemayehu/Recipe-app`
 
-* Deployment instructions
+### In your terminal, inside the project folder
 
-* ...
+`rails s`
+
+### Open `127.0.0.1:3000` on your web browser
+
+## Authors
+
+👤 **Tadesse Alemayehu**
+
+- GitHub: [@Tadess-Alemayehu](https://github.com/Tadesse-Alemayehu)
+- Twitter: [@TadesseWebDev](https://twitter.com/TadesseWebDev)
+- LinkedIn: [Tadesse Alemayehu](https://www.linkedin.com/in/tadesse-alemayehu-60141a221/)
+
+👤 **Mar Bautista**
+
+- GitHub: [marilirulita](https://github.com/marilirulita)
+- LinkedIn: [mar bautista](https://www.linkedin.com/in/marbautista/)
+- Twitter: [@marylirulita](https://twitter.com/marylirulita)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to all our code reviewers and partners at Microverse.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly make sure you add the **[LICENCE](https://github.com/microverseinc/readme-template/blob/master/MIT.md) MIT** file in your root directory in order to fix this broken link `[MIT](./LICENSE)` and make your README profesional.

Reply:
It is done!